10-std.tex

```diff
@@ -295,7 +295,79 @@ \subsection{Extensions}
 \section{Lambda}
 \label{std-Lambda}
 
+\define{Lambda}{define-lambda}{Lambda is a functional language concept within Haxe that allows you to apply a function to a set or \tref{custom iterators}{lf-iterators}. The Lambda class is a collection of functional methods in order to use functional-style programming with Haxe.}
 
+It is ideally used with \expr{using Lambda} (read \tref{Static Extension}{lf-static-extension}) and then acts as an extension to \type{Iterable} types. 
+
+On static platforms, working with the \type{Iterable} structure might be slower than performing the operations directly on known types, such as \type{Array} and \type{List}.
```

Comment:
I tried to talk about "targets" instead of "platforms" throughout the manual because I think it is more accurate.